utils_nlp/models/mtdnn/process_mtdnn.py

```diff
@@ -363,89 +362,98 @@ def fit(self, epochs=0):
                     logger.info(f"Saving mt-dnn model to {model_file}")
                     self.model.save(model_file)
 
-    def predict(self, epochs=0):
+            """Xiaodl: I move the save model func here since it makes more senses
+            that is save each checkpoint after the model training.
+            I further move the save function in prediction.
+            Alternatively, we need to refactor save function.
+            """
+            model_file = os.path.join(
+                self.output_dir, "model_{}.pt".format(epoch)
+            )
+            logger.info(f"Saving mt-dnn model to {model_file}")
+            self.model.save(model_file)
+
+    def predict(self, epoch=0):
```

Comment:
instead of using epoch. We need the model checkpoint where the default is the current model.